versioned/transfer/src/test/java/org/projectnessie/versioned/transfer/BaseExportImport.java

```diff
@@ -344,34 +347,34 @@ private static Hash commit10(VersionStore versionStore, int offset, BranchName b
   }
 
   List<KeyEntry> keys(VersionStore versionStore, Hash hash) {
-    try (Stream<KeyEntry> keys = versionStore.getKeys(hash)) {
-      return keys.collect(Collectors.toList());
+    try (PaginationIterator<KeyEntry> keys = versionStore.getKeys(hash, null)) {
+      return newArrayList(keys);
```

Comment:
Isn't this risky? If/when the store starts limiting responses, the list will be incomplete and there's no provision for fetching subsequent pages.

Reply:
It's test code